config/routes.rb

```diff
@@ -32,6 +32,9 @@
       resources :company_payment_methods, only: [:index]
       post '/invoices', to: 'invoices#create'
       resources :products, only: [:create, :update], param: :token
+      resources :subscriptions, only: %i[create]
+      get '/subscription/show/:company_token/:subscription_token', to: 'subscriptions#show', as: 'subscription_show'
+      get '/subscriptions/index/:company_token/:purchaser_token', to: 'subscriptions#index', as: 'subscription_index'
```

Comment:
O que acham de pensar em algo que possa melhorar a legibilidade dessas rotas tipo algo como:

```suggestion
      resources :company, param: :token do
        resources :subscriptions, param: :token, only: %i[index show]
      end
```

Será que daria certo?